{"uid":"f3a9a277e1b64a40","name":"test_navigate_back_to_start_page","fullName":"tests_all.TestNavigate#test_navigate_back_to_start_page","historyId":"a5b52b9a2ea7d1fcdb5e8cbe2ba3e0c7","time":{"start":1728831410210,"stop":1728831410210,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55bcd006c02a <unknown>\n#1 0x55bccfd525e0 <unknown>\n#2 0x55bccfd8a921 <unknown>\n#3 0x55bccfd862c5 <unknown>\n#4 0x55bccfdd2df6 <unknown>\n#5 0x55bccfdd2446 <unknown>\n#6 0x55bccfdc68c3 <unknown>\n#7 0x55bccfd946b3 <unknown>\n#8 0x55bccfd9568e <unknown>\n#9 0x55bcd0036a2b <unknown>\n#10 0x55bcd003a9b1 <unknown>\n#11 0x55bcd0023225 <unknown>\n#12 0x55bcd003b532 <unknown>\n#13 0x55bcd000838f <unknown>\n#14 0x55bcd005af28 <unknown>\n#15 0x55bcd005b0f3 <unknown>\n#16 0x55bcd006ae7c <unknown>\n#17 0x7f5ef1a9ca94 <unknown>\n#18 0x7f5ef1b29c3c <unknown>","statusTrace":"request = <SubRequest 'init_driver' for <Function test_navigate_back_to_start_page>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_driver(request):\n        # Инициализация драйвера\n>       driver = webdriver.Chrome()\n\nconftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py:66: in __init__\n    super().__init__(command_executor=executor, options=options)\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:212: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:299: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:354: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f4d2d666260>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"session not created\",\"message\":\"session not created: Chrome failed to sta...\\\\n#16 0x55bcd006ae7c \\\\u003Cunknown>\\\\n#17 0x7f5ef1a9ca94 \\\\u003Cunknown>\\\\n#18 0x7f5ef1b29c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\nE         (session not created: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x55bcd006c02a <unknown>\nE       #1 0x55bccfd525e0 <unknown>\nE       #2 0x55bccfd8a921 <unknown>\nE       #3 0x55bccfd862c5 <unknown>\nE       #4 0x55bccfdd2df6 <unknown>\nE       #5 0x55bccfdd2446 <unknown>\nE       #6 0x55bccfdc68c3 <unknown>\nE       #7 0x55bccfd946b3 <unknown>\nE       #8 0x55bccfd9568e <unknown>\nE       #9 0x55bcd0036a2b <unknown>\nE       #10 0x55bcd003a9b1 <unknown>\nE       #11 0x55bcd0023225 <unknown>\nE       #12 0x55bcd003b532 <unknown>\nE       #13 0x55bcd000838f <unknown>\nE       #14 0x55bcd005af28 <unknown>\nE       #15 0x55bcd005b0f3 <unknown>\nE       #16 0x55bcd006ae7c <unknown>\nE       #17 0x7f5ef1a9ca94 <unknown>\nE       #18 0x7f5ef1b29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:229: SessionNotCreatedException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"init_driver","time":{"start":1728831410210,"stop":1728831410391,"duration":181},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55bcd006c02a <unknown>\n#1 0x55bccfd525e0 <unknown>\n#2 0x55bccfd8a921 <unknown>\n#3 0x55bccfd862c5 <unknown>\n#4 0x55bccfdd2df6 <unknown>\n#5 0x55bccfdd2446 <unknown>\n#6 0x55bccfdc68c3 <unknown>\n#7 0x55bccfd946b3 <unknown>\n#8 0x55bccfd9568e <unknown>\n#9 0x55bcd0036a2b <unknown>\n#10 0x55bcd003a9b1 <unknown>\n#11 0x55bcd0023225 <unknown>\n#12 0x55bcd003b532 <unknown>\n#13 0x55bcd000838f <unknown>\n#14 0x55bcd005af28 <unknown>\n#15 0x55bcd005b0f3 <unknown>\n#16 0x55bcd006ae7c <unknown>\n#17 0x7f5ef1a9ca94 <unknown>\n#18 0x7f5ef1b29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/Ykassa/Ykassa/conftest.py\", line 8, in init_driver\n    driver = webdriver.Chrome()\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 212, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 299, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('init_driver', 'base_url')"},{"name":"suite","value":"tests_all"},{"name":"subSuite","value":"TestNavigate"},{"name":"host","value":"fv-az789-40"},{"name":"thread","value":"2538-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests_all"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"89c4241c104d6ab0","reportUrl":"https://DyatlovaTatyana.github.io/Ykassa/1//#testresult/89c4241c104d6ab0","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: Chrome failed to start: exited normally.\n  (session not created: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55e94ba0302a <unknown>\n#1 0x55e94b6e95e0 <unknown>\n#2 0x55e94b721921 <unknown>\n#3 0x55e94b71d2c5 <unknown>\n#4 0x55e94b769df6 <unknown>\n#5 0x55e94b769446 <unknown>\n#6 0x55e94b75d8c3 <unknown>\n#7 0x55e94b72b6b3 <unknown>\n#8 0x55e94b72c68e <unknown>\n#9 0x55e94b9cda2b <unknown>\n#10 0x55e94b9d19b1 <unknown>\n#11 0x55e94b9ba225 <unknown>\n#12 0x55e94b9d2532 <unknown>\n#13 0x55e94b99f38f <unknown>\n#14 0x55e94b9f1f28 <unknown>\n#15 0x55e94b9f20f3 <unknown>\n#16 0x55e94ba01e7c <unknown>\n#17 0x7f152fc9ca94 <unknown>\n#18 0x7f152fd29c3c <unknown>","time":{"start":1728830429951,"stop":1728830429951,"duration":0}}]},"tags":["@pytest.mark.usefixtures('init_driver', 'base_url')"]},"source":"f3a9a277e1b64a40.json","parameterValues":[]}